### ISSUE TITLE
Question: Cannot find module 'playwright-bdd/decorators' in version 7.4.0 

### ISSUE THREAD

**Issue by mothuc** [vitalets/playwright-bdd#222]
I'm experiencing an issue with TypeScript when trying to import the playwright-bdd/decorators module. The error message I'm getting is:
```
Cannot find module 'playwright-bdd/decorators' or its corresponding type declarations.
  There are types at '/home/thuc/Projects/pw-bdd-mwk-hk/node_modules/playwright-bdd/dist/decorators.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.ts(2307)
```
Here is fixture.ts:
![image](https://github.com/user-attachments/assets/7b438e44-ccb2-4e9d-a488-03c926768599)

Error when import:
![image](https://github.com/user-attachments/assets/604ea3bf-9d34-4be6-8efc-e9e82bfe0989)

**Comment by vitalets** [comment]
Is it suitable fro you project to set 'moduleResolution' to 'node16' or 'nodenext' in tsconfig.json?

**Comment by vitalets** [comment]
Fixed in v7.4.2 for projects with `module` = `commonjs`. 
Could you check on your side?

Note: changing `module` to `nodenext` in tsconfig is more preferable and [recommended](https://www.typescriptlang.org/docs/handbook/modules/reference.html#summary) by TypeScript docs.

**Comment by mothuc** [comment]
Its work. Thanks bro
